web/package.json

```diff
@@ -19,6 +19,7 @@
     "i18next-browser-languagedetector": "^6.1.4",
     "i18next-http-backend": "^1.4.0",
     "js-base64": "^3.7.2",
+    "nanoid": "^4.0.0",
```

Comment:
Adding a new library for generating a random string is insane :-) That can easily be done in JS without a new lib.

Reply:
Dropped the dependency